src/commands/Games/coinflip.ts

```diff
@@ -0,0 +1,56 @@
+import { CommandStore, KlasaMessage } from 'klasa';
+import { SkyraCommand } from '../../lib/structures/SkyraCommand';
+import { UserSettings } from '../../lib/types/settings/UserSettings';
+import { MessageEmbed } from 'discord.js';
+import { getColor, cleanMentions } from '../../lib/util/util';
+
+const enum CoinTypes { Heads, Tails }
+export default class extends SkyraCommand {
+
+	private readonly cdnTypes = ['heads', 'tails'] as const;
+
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			aliases: ['cf'],
+			bucket: 2,
+			cooldown: 7,
+			description: language => language.tget('COMMAND_COINFLIP_DESCRIPTION'),
+			extendedHelp: language => language.tget('COMMAND_COINFLIP_EXTENDED'),
+			requiredPermissions: ['ATTACH_FILES'],
```

Comment:
I'm pretty sure you don't need this with embeds that set a thumbnail from an url

Reply:
The required permission in this case is `EMBED_LINKS`